New blog post with updates on the evolution of XRPL (#2864)
XRPLF/xrpl-dev-portal: blog/2024/a-new-era-for-the-xrp-ledger.md

seo:
    title: The Evolution of the XRPL Foundation - A New Era for XRPL
    description: Discover the latest updates on the new XRPL Foundation’s incorporation, objectives, governance, and structure.
 

labels:
    - General
markdown:
    editPage:
        hide: true
---
# The Evolution of the XRPL Foundation: A New Era for XRPL

_By XRPL Foundation, XRPL Labs, XRPL Commons, and Ripple_

Earlier this year a transformative evolution of the XRPL Foundation was [announced](./evolving-the-xrp-ledger.md). Organizations including XRPL Commons, XRPL Labs, Ripple, and XAO DAO, a community DAO, have started the process of establishing a new, inclusive, and visible XRPL Foundation to provide long-term security, coordination, and effective, decentralized contributions to the XRPL, while at the same time promoting enhanced collaboration between community members.

Since the initial announcement, the first steps towards the creation of the new XRPL Foundation have commenced. This article delves into the recent developments, exploring the new Foundation's incorporation, objectives, governance, and structure.

## A Significant Milestone: The Incorporation of the New Foundation

> Today, November 26th, 2024, a milestone has been reached as the incorporation documents for the new XRPL Foundation were filed in France today. This non-profit association, governed by French 1901 law, has been filed under the name 'XRP Ledger Foundation'. The decision to incorporate in France aligns with the original objectives laid out in the [XRPL.org post](./evolving-the-xrp-ledger.md) on August 10th, 2024, and marks a new chapter in the Foundation's history. The filing is available [here](https://www.journal-officiel.gouv.fr/pages/associations-detail-annonce/?q.id=id:202400481665).

Reviewer: Suggestion to use more descriptive link text:

```suggestion
Today, November 26th, 2024, a milestone has been reached as the incorporation documents for the new XRPL Foundation were filed in France today. This non-profit association, governed by French 1901 law, has been filed under the name 'XRP Ledger Foundation'. The decision to incorporate in France aligns with the original objectives laid out in the [XRPL.org post](./evolving-the-xrp-ledger.md) on August 10th, 2024, and marks a new chapter in the Foundation's history. You can [access the official filing online](https://www.journal-officiel.gouv.fr/pages/associations-detail-annonce/?q.id=id:202400481665).
```

The XRPL Foundation will operate under French law, with the Paris Court of Appeals having exclusive jurisdiction over disputes. As part of the incorporation of the new Foundation, a comprehensive set of bylaws has been established to ensure effective governance. The bylaws provide a framework for decision-making, member roles, and operational procedures.

> The full copy of the bylaws in both French and English are available [here](/2024-11-13-XRPLedger-Foundation-ByLaws-(Statuts).pdf).

Reviewer: It's bad practice and bad SEO to use "here" as link text. Also, I prefer to give warnings when linking to PDFs, since some browsers/devices may automatically download a PDF and open it in another application:

```suggestion
You can also view [the full copy of the bylaws in both French and English (PDF)](/2024-11-13-XRPLedger-Foundation-ByLaws-(Statuts).pdf).
```

## The New Foundation’s Purpose 

The new XRPL Foundation’s purpose is clear: to promote and facilitate a secure, resilient, inclusive, and thriving XRPL community which empowers users and developers to confidently build decentralized applications, financial tools, and infrastructure.

It will achieve this by focusing on two key objectives:

1. To promote and facilitate the community and operations of the XRPL.
2. To support the XRP Ledger, ensuring it functions in an open, decentralized, and transparent manner.

## Responsibilities and Areas of Focus

> To fulfill its purpose and meet its objectives, the Foundation has outlined an extensive list of areas that it will focus on:

Reviewer: Nothing to do here but smile wryly at the oxymoron of "focusing" on an "extensive list of areas".
